tastytrade/instruments.py

```diff
@@ -185,6 +247,12 @@ def get_equity(cls, session: Session, symbol: str) -> 'Equity':
         return cls.from_dict(data)
 
 
+@dataclass
+class ActiveEquitiesResponse:
```

Comment:
This class shouldn't exist, let's just return the list of equities from the `get_active_equities` call.